prac_05/word_occurrences.py

```diff
@@ -0,0 +1,35 @@
+"""
+Time estimate: 45 minutes
+"""
+
```

Comment:
If there is a function, better to make a main function also. However, I don't think you are asked to make a separate function here.